package.json

```diff
@@ -71,7 +71,7 @@
     "@babel/preset-env": "^7.14.5",
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.14.5",
-    "@department-of-veterans-affairs/component-library": "^11.3.0",
+    "@department-of-veterans-affairs/component-library": "11.5.4",
```

Comment:
Just want to make sure if `^` was purposely omitted.

Reply:
We can add it back if you want, just got removed for testing purposes really.